pkg/pquerier/sqlParser.go

```diff
@@ -43,17 +43,27 @@ func ParseQuery(sql string) (*SelectParams, string, error) {
 
 			switch expr := col.Expr.(type) {
 			case *sqlparser.FuncExpr:
-				cc := expr.Exprs[0].(*sqlparser.AliasedExpr).Expr.(*sqlparser.ColName)
-				currCol.Function = sqlparser.String(expr.Name)
-				currCol.Interpolator = removeComma(sqlparser.String(cc.Qualifier.Name)) // Some of the interpolators are parsed with a `
-				currCol.Metric = sqlparser.String(cc.Name)
+				switch firstExpr := expr.Exprs[0].(type) {
+				case *sqlparser.AliasedExpr:
+					cc := firstExpr.Expr.(*sqlparser.ColName)
+					currCol.Function = sqlparser.String(expr.Name)
+					currCol.Interpolator = removeComma(sqlparser.String(cc.Qualifier.Name)) // Some of the interpolators are parsed with a `
```

Comment:
`removeComma`->`removeTicks`

Reply:
https://help.github.com/articles/creating-and-highlighting-code-blocks/#fenced-code-blocks